02-peer-protocol.md

```diff
@@ -1033,6 +1036,21 @@ A sending node:
   - MUST increase the value of `id` by 1 for each successive offer.
   - if it is relaying a payment inside a blinded route:
     - MUST set `blinding_point` (see [Route Blinding](04-onion-routing.md#route-blinding))
+  - if it is the original source of the HTLC:
+    - if it does not expect immediate fulfillment upon receipt by the 
+      final destination: 
+      - SHOULD set `endorsed` to `0`.
+    - otherwise:  
+      - SHOULD set `endorsed` to `1`. 
+      - MAY choose to set `endorsed` to `0` to mimic endorsement patterns of 
+        HTLCs it has forwarded.
```

Comment:
This is when the sender is a hop of the route and it will just forward the value received without modifying it, right?

Reply:
Assume that Alice usually routes payments that come in with  `endorsed` set to `0`. Then, if she sends her own payment with  `endorsed` set to `1`, her neighbor might guess that she is the sender.
To avoid this, Alice can choose occasionally to set  `endorsed` to `0` even for payments that she is sending herself.

Reply:
> This is when the sender is a hop of the route and it will just forward the value received without modifying it, right?

This is nested under "if it is the original source of the HTLC:" so the directive only applies to the original sender (Alice, in Clara's example).

Reply:
Ah I see now, thanks for the clarification